plugins/catalog-backend/src/database/operations/stitcher/getDeferredStitchableEntities.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { durationToMilliseconds, HumanDuration } from '@backstage/types';
+import { Knex } from 'knex';
+import { DateTime } from 'luxon';
+import { timestampToDateTime } from '../../conversion';
+import { DbRefreshStateRow } from '../../tables';
+
+// TODO(freben): There is no retry counter or similar. If items start
+// perpetually crashing during stitching, they'll just get silently retried over
+// and over again, for better or worse. This will be visible in metrics though.
```

Comment:
would it be visible in any logs?

Reply:
Yes. The default implementation of the stitch tracker (that also emits metrics) does an error level log when the stitcher throws.

https://github.com/backstage/backstage/blob/8d756968f9c8676c391868a5a9aae931994c877b/plugins/catalog-backend/src/stitching/DefaultStitcher.ts#L188

https://github.com/backstage/backstage/blob/8d756968f9c8676c391868a5a9aae931994c877b/plugins/catalog-backend/src/stitching/progressTracker.ts#L98